core/src/main/java/org/frankframework/jdbc/JdbcTableMessageBrowser.java

```diff
@@ -154,9 +150,9 @@ protected String getSelectListQuery(IDbmsSupport dbmsSupport, Date startTime, Da
 		}
 		if(order == SortOrder.NONE) { //If no order has been set, use the default (DESC for messages and ASC for errors)
 			order = getOrder();
-			if (order == null) {
-				order = SortOrder.ASC;
-			}
+		}
+		if (order == null) {
+			order = SortOrder.ASC;
```

Comment:
This change prevents an NPE when the `order` argument was `null` -- which it was when logging queries from the configuration of JdbcTransactionalStorage.